modules/ascat/meta.yml

```diff
@@ -0,0 +1,88 @@
+name: ascat
+description: copy number profiles of tumour cells.
+keywords:
+  - sort
+tools:
+  - ascat:
+      description: ASCAT is a method to derive copy number profiles of tumour cells, accounting for normal cell admixture and tumour aneuploidy. ASCAT infers tumour purity (the fraction of tumour cells) and ploidy (the amount of DNA per tumour cell), expressed as multiples of haploid genomes from SNP array or massively parallel sequencing data, and calculates whole-genome allele-specific copy number profiles (the number of copies of both parental alleles for all SNP loci across the genome).
+      homepage: None
+      documentation: None
+      tool_dev_url: https://github.com/Crick-CancerGenomics/ascat
+      doi: "10.1093/bioinformatics/btaa538"
+      licence: ['GPL v3']
+
+input:
+    - args:
+    type: map
+    description: Groovy Map containing tool parameters. MUST follow the structure/keywords below and be provided via modules.config. <optional> parameters can be removed from the map, if they are not set. For default values, please check the documentation above. ```
+      {
+        [
+          "gender": "XX",
+          "genomeVersion": "hg19"
+          "purity": <optional>,
+          "ploidy": <optional>,
+          "gc_files": <optional>,
+          "minCounts": <optional>,
+          "chrom_names": <optional>,
+          "min_base_qual": <optional>,
+          "min_map_qual": <optional>,
+          "ref_fasta": <optional>,
+          "skip_allele_counting_tumour": <optional>,
+          "skip_allele_counting_normal": <optional>
+        ]
+      }
+      ```
```

Comment:
seems like you are missing the opening quotes

Reply:
ups I see they are at the end of the line above, I think markdown expects them to in a new line surrounded by empty lines